src/Player/PlayerActorBase.h

```diff
@@ -41,5 +42,11 @@ class PlayerActorBase : public al::LiveActor, public IUsePlayerHack {
     virtual void movement() override;
     virtual void checkDeathArea();
     virtual void sendCollisionMsg();
-    virtual void receivePushMsg(al::SensorMsg const*,al::HitSensor *,al::HitSensor *,float);
+    virtual void receivePushMsg(const al::SensorMsg*, al::HitSensor*, al::HitSensor*, float);
 };
+
+class IUsePlayerCollision {
+public:
+    // TODO: Does this return PlayerCollider or something else?
```

Comment:
You should probably research that in this PR when adding the class - seems important enough to not be ignored here.

Reply:
Looks like you have it as `PlayerCollider *` in your repo. Will change it to that as I can't find another answer.